mac

```diff
@@ -176,6 +178,8 @@ brew_tap 'caskroom/cask'
 brew_install_or_upgrade 'brew-cask'
 brew_tap 'caskroom/versions'
 
+brew_cask_install 'alfred'
+brew cask install 'dash'
```

Comment:
The `Spotlight` came with Yosemite covers pretty much all functionalities of `Alfred`. The one of the few differences is you can type some system commands like `restart` on `Alfred`. 

As @grantovich's comment, I think it is a personal preference too.